src/Uno.UI.Tests/Windows_Globalization/When_Calendar.cs

```diff
@@ -0,0 +1,692 @@
+
+			SUT.SetDateTime(date);
+
+			mutator?.Invoke(SUT);
+
+			using (new AssertionScope("Calendar Properties"))
+			{
+				SUT.Day.Should().Be(day, "day");
+				SUT.Month.Should().Be(month, "month");
+				SUT.Year.Should().Be(year, "year");
+				SUT.Hour.Should().Be(hours, "hours");
+				SUT.Minute.Should().Be(minutes, "minutes");
+				SUT.Second.Should().Be(seconds, "seconds");
+				SUT.Nanosecond.Should().Be(milliseconds * 1000, "milliseconds");
+				SUT.DayOfWeek.Should().Be(dayOfWeek, "dayOfWeek");
+				SUT.Era.Should().Be(era, "era");
+				SUT.FirstDayInThisMonth.Should().Be(firstDayInThisMonth, "firstDayInThisMonth");
+				SUT.FirstEra.Should().Be(firstEra, "firstEra");
+				SUT.FirstHourInThisPeriod.Should().Be(firstHourInThisPeriod, "firstHourInThisPeriod");
+				SUT.FirstMinuteInThisHour.Should().Be(firstMinuteInThisHour, "firstMinuteInThisHour");
+				SUT.FirstMonthInThisYear.Should().Be(firstMonthInThisYear, "firstMonthInThisYear");
+				SUT.FirstPeriodInThisDay.Should().Be(firstPeriodInThisDay, "firstPeriodInThisDay");
+				SUT.FirstSecondInThisMinute.Should().Be(firstSecondInThisMinute, "firstSecondInThisMinute");
+				SUT.FirstYearInThisEra.Should().Be(firstYearInThisEra, "firstYearInThisEra");
+				SUT.IsDaylightSavingTime.Should().Be(isDaylightSavingTime, "isDaylightSavingTime");
+				SUT.Languages.Should().HaveCount(1, "languages count");
+				SUT.Languages.Should().HaveElementAt(0, culture, "culture");
+				SUT.LastDayInThisMonth.Should().Be(lastDayInThisMonth, "lastDayInThisMonth");
+				SUT.LastEra.Should().Be(lastEra, "lastEra");
+				SUT.LastHourInThisPeriod.Should().Be(lastHourInThisPeriod, "lastHourInThisPeriod");
+				SUT.LastMinuteInThisHour.Should().Be(lastMinuteInThisHour, "lastMinuteInThisHour");
+				SUT.LastMonthInThisYear.Should().Be(lastMonthInThisYear, "lastMonthInThisYear");
+				SUT.LastPeriodInThisDay.Should().Be(lastPeriodInThisDay, "lastPeriodInThisDay");
+				SUT.LastSecondInThisMinute.Should().Be(lastSecondInThisMinute, "lastSecondInThisMinute");
+				SUT.LastYearInThisEra.Should().Be(lastYearInThisEra, "lastYearInThisEra");
+				SUT.NumberOfDaysInThisMonth.Should().Be(numberOfDaysInThisMonth, "numberOfDaysInThisMonth");
+				SUT.NumberOfEras.Should().Be(numberOfEras, "numberOfEras");
+				SUT.NumberOfHoursInThisPeriod.Should().Be(numberOfHoursInThisPeriod, "numberOfHoursInThisPeriod");
+				SUT.NumberOfMinutesInThisHour.Should().Be(numberOfMinutesInThisHour, "numberOfMinutesInThisHour");
+				SUT.NumberOfMonthsInThisYear.Should().Be(numberOfMonthsInThisYear, "numberOfMonthsInThisYear");
+				SUT.NumberOfPeriodsInThisDay.Should().Be(numberOfPeriodsInThisDay, "numberOfPeriodsInThisDay");
+				SUT.NumberOfSecondsInThisMinute.Should().Be(numberOfSecondsInThisMinute, "numberOfSecondsInThisMinute");
+				SUT.NumberOfYearsInThisEra.Should().Be(numberOfYearsInThisEra, "numberOfYearsInThisEra");
+				// SUT.NumeralSystem.Should().Be(numeralSystem, "numeralSystem");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6178917)